opentracing-api/src/main/java/io/opentracing/propagation/Adapters.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016-2017 The OpenTracing Authors
```

Comment:
nit: as this is new code, (c) is only 2017 (same in other files)

Reply:
:) Had missed that, will fix.

Reply:
Oh, btw, if I use `2017` alone the build fails. Guess we need to issue a PR for fixing this part in pom.xml ;)